server/views/pages/create/confirmation.njk

```diff
@@ -8,20 +9,29 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             {{ govukPanel({
-                titleText: "Licence conditions for " + offender.name + " sent"
+              titleText: "Licence conditions for " + licence.forename + " " + licence.surname + " sent"
             }) }}
 
             <h2 class="govuk-heading-m">What happens next</h2>
-            <p class="govuk-body">
-                We have sent the licence conditions to {{ offender.prison }} for approval.
+            <p class="govuk-body" id="sent-to">
+                We have sent the licence conditions to {{ licence.prisonDescription }} for approval.
             </p>
             <p class="govuk-body">
                 You will get an email notification when they are approved.
             </p>
 
             <h2 class="govuk-heading-m">Give us your feedback</h2>
             <p class="govuk-body">
-                <a href="#" class="govuk-link">What did you think of this service?</a> (takes 30 seconds)
+                <a href="{{ exitSurveyLink }}" target="_blank" id="exit-survey" class="govuk-link">What did you think of this service?</a> (takes 30 seconds)
```

Comment:
When we are linking to an external site, we should use `rel="noreferrer noopener"` attribute for security

https://web.dev/external-anchors-use-rel-noopener/

Reply:
Thanks. Updated now.